test_scripts/API/IsReady/TTS_IsReady/NotUsed/ATF_TTS_IsReady_NotRespond_Split_RPC_APPLINK_25139_RETRY.lua

```diff
@@ -417,7 +417,7 @@ function Test:initHMI_onReady_TTS_IsReady(case)
 	
 	
 	ExpectRequest("TTS.GetCapabilities", true, {
-		speechCapabilities = { "TEXT", "SAPI_PHONEMES", "LHPLUS_PHONEMES", "PRE_RECORDED", "SILENCE" },
+		speechCapabilities = { "TEXT", "SAPI_PHONEMES", "LHPLUS_PHONEMES", "PRE_RECORDED", "SILENCE", "FILE" },
```

Comment:
is it possible to define a speechCapabilities in common code and reference it everywhere ?